<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFrontend</class>
 <widget class="QMainWindow" name="QFrontend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFrontend</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="6" rowspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QPushButton" name="quit">
        <property name="text">
         <string>quit context</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="dwTrafficTable"/>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="ontID">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="port"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btLogin">
        <property name="text">
         <string>Conectar</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Slot</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Serial number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Frame</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btCmd">
        <property name="text">
         <string>Send Cmd</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Custom value</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Port (GPON)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>ONT ID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>OLT VLAN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="priority"/>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>PIR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QSpinBox" name="upTrafficTable"/>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Up TTable IP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Priority</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QSpinBox" name="ontVLAN"/>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="gemPort"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>ONT VLAN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>GEM Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QTypeSpinBox" name="type">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QSpinBox" name="oltVLAN"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Dw TTable IP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QComboBox" name="command"/>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="slot"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="sendCMD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="value">
        <property name="maximum">
         <number>512</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="cir">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="frame"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="serial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="cmd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="pir">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>CIR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Max speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="3">
       <widget class="QSpeedSpinBox" name="max">
        <property name="minimum">
         <number>128</number>
        </property>
        <property name="maximum">
         <number>1048576</number>
        </property>
        <property name="singleStep">
         <number>64</number>
        </property>
        <property name="value">
         <number>1048576</number>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QSpeedSpinBox" name="assured">
        <property name="minimum">
         <number>128</number>
        </property>
        <property name="maximum">
         <number>1048576</number>
        </property>
        <property name="singleStep">
         <number>64</number>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QSpeedSpinBox" name="fix">
        <property name="minimum">
         <number>128</number>
        </property>
        <property name="maximum">
         <number>1048576</number>
        </property>
        <property name="singleStep">
         <number>64</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Assured speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Fix Speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="2" colspan="2">
       <widget class="QCheckBox" name="bandwitdh_compensation">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Bandwidth compensation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Username</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>OLT FQDN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="leFQDN"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QSpinBox" name="sbPort">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="6">
     <widget class="QPlainTextEdit" name="telnetOut">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="3">
     <widget class="QLineEdit" name="leUPass"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leUName"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTypeSpinBox</class>
   <extends>QSpinBox</extends>
   <header>QOLTCommands.h</header>
  </customwidget>
  <customwidget>
   <class>QSpeedSpinBox</class>
   <extends>QSpinBox</extends>
   <header>QOLTCommands.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leFQDN</tabstop>
  <tabstop>sbPort</tabstop>
  <tabstop>leUName</tabstop>
  <tabstop>leUPass</tabstop>
  <tabstop>telnetOut</tabstop>
  <tabstop>btLogin</tabstop>
  <tabstop>quit</tabstop>
  <tabstop>btCmd</tabstop>
  <tabstop>cmd</tabstop>
  <tabstop>command</tabstop>
  <tabstop>sendCMD</tabstop>
  <tabstop>serial</tabstop>
  <tabstop>value</tabstop>
  <tabstop>name</tabstop>
  <tabstop>frame</tabstop>
  <tabstop>cir</tabstop>
  <tabstop>slot</tabstop>
  <tabstop>pir</tabstop>
  <tabstop>port</tabstop>
  <tabstop>priority</tabstop>
  <tabstop>ontID</tabstop>
  <tabstop>upTrafficTable</tabstop>
  <tabstop>dwTrafficTable</tabstop>
  <tabstop>ontVLAN</tabstop>
  <tabstop>gemPort</tabstop>
  <tabstop>type</tabstop>
  <tabstop>oltVLAN</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
